flows/flows.go

```diff
@@ -89,6 +92,10 @@ func (f *Flow) String() string {
 	}
 
 	var actionSet []string
+	if f.ethDest != "" {
+		actionSet = append(actionSet, fmt.Sprintf("set_field:%s->eth_dst", f.ethDest))
```

Comment:
Is this the same as `mod_dl_dst`?

Reply:
yes, set_field is more consistent AKAIK and works across all layers on the defined match keys so either to remember. Also, if you look at the specification there is no mention of `mod_dl_dst`. But to answer your question, they should do the same thing.

Reply:
gotcha, thanks